RecoLocalCalo/HGCalRecAlgos/src/HGCal3DClustering.cc

```diff
@@ -102,9 +102,13 @@ std::vector<reco::HGCalMultiCluster> HGCal3DClustering::makeClusters(const reco:
 
       }
       if( temp.size() > minClusters ) {
+	math::XYZPoint position = clusterTools->getMultiClusterPosition(temp);
+	if (abs(position.z()) < 100) continue;
```

Comment:
Can the magic value 100 be made into an explanatory const value? Is it derivable from geometry information or just an arbitrary cut?